WIP: [Closed Beta] Custom Views initial technical documentation (#3289)
commercetools/merchant-center-application-kit: website-custom-views/src/content/api-reference/custom-view-config.mdx

  }
}
```

## `labelAllLocales`

An optional list of localized fields `{ locale, value }` with translated labels for each custom view locale, available in the user profile.

These values will be used when presenting the user with a list of available Custom Views in the Merchant Center views.
If they are not configured, the `name` values will be used instead.

```json
{
  "labelAllLocales": [
    { "locale": "en", "value": "The Avengers" },
    { "locale": "de", "value": "Die Avengers" }
  ]
}
```

## `type`

The required type of Custom View.
Currently we only support the `CustomPanel` value.

```json
{
  "type": "CustomPanel"
}
```

## `typeSettings`

Depending on the Custom View type, different settings might be required.

## `typeSettings.size`

In case the `CustomPanel` type is used, the `size` property is required. It defines the size of the Custom View panel.

Supported values are: `SMALL`, `LARGE`.

```json
{
  "type": "CustomPanel",
  "typeSettings": {
    "size": "SMALL"
  }
}
```

## `locators`

Since Custom Views are rendered in the context of native Merchant Center applications, you need to set in which specific views the Custom View should be rendered.

> This configuration property is a list of codes which represent every Merchant Center view where the Custom View can be rendered.

Reviewer: ```suggestion
This configuration property is a list of identifiers of Merchant Center locations where the Custom View is allowed to be rendered.
```

Author: Update here: 558993f8b1746e3bd9f8435c3f961a39c118c7df

```json
{
  "locators": ["products.product_details.general"]
}
```

> The locator codes follow the pattern `<application>.<view>.<subview>`.
> The following list contains all the currently available locators:

Reviewer: ```suggestion
A locator follows the pattern `<application>.<view>.<subview>`.

The following list contains all the currently available locators:
```

Author: Update here: 558993f8b1746e3bd9f8435c3f961a39c118c7df
